Solutions/Menes.Specs/Features/AccessControlPolicy.feature

```diff
@@ -9,8 +9,6 @@ Scenario Outline: The request details are passed to all policies
     Then each policy should receive a path of '<path>'
     And each policy should receive an operationId of '<operationId>'
     And each policy should receive an HttpMethod of '<httpMethod>'
-    And each policy should receive the ClaimsPrincipal attached to the context
```

Comment:
I thought access to the `ClaimsPrincipal` is fundamental to the usefulness of access control policies. I guess the idea is that now these things are supposed to be accessed via DI, having been injected into the request scope by a scope builder?

But do we have any tests verifying that an access control policy will indeed be able to access per-scope dependencies?
Instead of just removing these parts of the tests, wouldn't it be useful to replace them with tests that verify that each policy is able to receive objects produced by scope builders working with this request? (Given that the claims access policy, which I think is the only meaningful implementation of this that we have, depends on the ClaimsPrincipal, I'd have been inclined to leave this particular test clause in, and just to change the setup so that the expected `ClaimsPrincipal` is fed into the system in the new way, by adding a suitable scope provider to the setup.)

Reply:
This is what I would describe as "something of an oversight"!

Reply:
I intended to wire this all back up... And will do so. C.f. next comment.